src/components/Inbox/index.jsx

```diff
@@ -0,0 +1,33 @@
+const Inbox = ({name, date, subject, body, read}) => {
```

Comment:
This does not seem to be 'an `Inbox`', it seems to be information about a particular email.